tox.ini

```diff
@@ -74,7 +74,8 @@ deps =
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
     pip-sync requirements/tests.txt requirements/pandapower.txt
-    pip install -e .
+	# https://stackoverflow.com/a/65147477/14830405
+    pip install -e . --use-deprecated=legacy-resolver
 	pip uninstall -y d3a-interface
```

Comment:
The lines here are not aligned, this is probably due to using both spaces and tabs, please fix